README.md

```diff
@@ -30,25 +30,19 @@
 
 ## About
 
-This repository is where the IOTA Foundation's Research Department tests the Coordicide modules to study and evaluate their performance.
+This repository, called GoShimmer, is where the IOTA Foundation's Research Department tests the IOTA 2.0 modules to study and evaluate their performance.
 
-The aim of this open repository is to give the community the opportunity to follow developments, take part in testing, and learn  more about [Coordicide](https://coordicide.iota.org/).
+GoShimmer is first and foremost a research prototype. As such, braking changes can often happen. We invite researchers and developers to make use of this project as you see fit. Running experiments, test out new ideas, build PoC are all very welcome initiatives.
 
-For a documentation, including tutorials and resources, we refer to the [Documentation](http://goshimmer.docs.iota.org/) (Github [link](https://github.com/iotaledger/goshimmer/blob/develop/docs/SUMMARY.md)).
-
-**Note:** You can find details about future development plans on our [roadmap](https://roadmap.iota.org).
+For a documentation, including tutorials and resources, we refer to the [Documentation](http://goshimmer.docs.iota.org/).
```

Comment:
i am not sure it is a good idea to remove the Github link. For example it was useful for when we have mdbook not picking up all links.